resource-timing/iframe-bad-redirect.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test the sequence of events when reporting iframe timing.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+    promise_test(async t => {
+        const frame = document.createElement('iframe');
+        t.add_cleanup(() => frame.remove());
+        await new Promise(resolve => {
+            frame.addEventListener('load', resolve);
+            frame.src = 'resources/bad-redirect.py';
```

Comment:
Nit: Maybe rename to redirect-without-location.py or add a comment as to why this redirect is bad?